src/nIsForName.js

```diff
@@ -22,7 +22,10 @@
  * @return {string[]}
  */
 function nIsForName(names) {
-  // write code here
+  const startsWithN = names
+    .filter(fullName => (fullName.split(' ')
+      .some(name => name[0] === 'N')));
```

Comment:
move the magic string to constant

Reply:
you mean separate this into steps? otherwise it wont pas lint test, what exactly should i do?

Reply:
magic string is hardcoded 'N', but ok, approved

Reply:
gotcha, thanks